### PR TITLE
Updates README.md to include libraries used

### LINKED ISSUE #226: Update README.md to include libraries used and building guideline
Sub issue for #176 

Expected:
1. Include libraries used in the project
2. Add development guideline including project setup

### PR DESCRIPTION
**Pull request by mejariamol**
- Also adds development guideline including project setup

fix #226

### CODE REVIEW

**Comment by mejariamol** [comment]
https://github.com/mejariamol/open-event-orga-app/blob/readme/README.md

@iamareebjamal please review.

**Comment by mejariamol** [comment]
@iamareebjamal Added via #228

**Comment by iamareebjamal** [comment]
@arp95 @Shailesh351 @rohanagarwal94 Please review

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ![Open Event Frontend](https://storage.googleapis.com/eventyay.com/assets/branding/organizer_app_branding.png)
```
Comment (iamareebjamal) [inline README.md]: Change this to Orga App please

Comment (mejariamol) [inline README.md]: done :+1:

**Pull request merged**